src/screens/StudentGroup.tsx

```diff
@@ -80,12 +79,18 @@ const StudentGroup: React.FC = () => {
   const updateBatch = useFirestore().batch()
 
   const resetSelectedStatus = () => {
+    const orderArray: number[] = []
+    for (let i = 1; i <= studentsInThisStudentGroupDocuments.length; i++) {
+      orderArray[i - 1] = i
+    }
+    console.log(orderArray)
```

Comment:
this feels a little odd, what is the purpose of this?

Reply:
I'm using that for loop to populate an array with numbers from 1 to however many kids are in the group. Then I'm randomly selecting a number, assigning it to an "order" property on each studentInStudentGroup object and splicing it from the array so I can keep track of which numbers have already been assigned. I'm doing this since Stephanie wants to know the order of students coming up. So the students get shuffled whenever the unselected students run out, and then the students are displayed in the order they'll be picked in. So a random order but displayed so the teacher can see who's coming. In full screen mode it will just show the name so the kids won't necessarily know the order. Does that make sense?